src/encodeOpReturnData.h

```diff
@@ -6,8 +6,8 @@
 #include <iomanip>
 
 // The data returned by OP_RETURN needs to be UTF-8, hex-encoded. Assuming UTF-8 for now.
-std::string encodeOpReturnData(const std::string &data) {
-    //TODO sanitize input
+inline std::string encodeOpReturnData(const std::string &data) {
+    //TODO sanitize input? what data is acceptable--just url-valid data?
```

Comment:
For purposes of censorship resistance, any data is acceptable.  OP_RETURN limits have changed several times, but the current limit is 83 bytes of binary data.

Formats that we'll ask resolvers to understand will include a plaintext URL and an IPFS.io hash (which may be converted to a URL through online trial and error, but which otherwise looks like common blockchain data storage, and which might not yet be posted to IPFS when the miners generate their blocks).  

We'll need some other extensibility registry once we go beyond IPFS.